db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_08_30_173705) do
+ActiveRecord::Schema.define(version: 2018_11_12_100217) do
 
-  create_table "alerts", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "alerts", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4", force: :cascade do |t|
```

Comment:
lots of extraneous changes to the DB here. Only the `create_table "training_modules"` block and the version bump should be included. (Also, the default charset is still just utf8 in production, and we should probably keep that consistent with the new table as well.)